spec/support/features/document_capture_step_helper.rb

```diff
@@ -1,20 +1,54 @@
 module DocumentCaptureStepHelper
+  def wait_for_form_page_load
+    wait_for_content_to_disappear do
+      expect(page).not_to have_content(t('doc_auth.headings.interstitial'), wait: 10)
+    end
+  end
+
   def submit_images
     click_on 'Submit'
 
     # Wait for the the loading interstitial to disappear before continuing
-    wait_for_content_to_disappear do
-      expect(page).not_to have_content(t('doc_auth.headings.interstitial'), wait: 10)
-    end
+    wait_for_form_page_load
   end
 
   def continue_doc_auth_form
     click_on 'Continue'
 
     # Wait for the the loading interstitial to disappear before continuing
-    wait_for_content_to_disappear do
-      expect(page).not_to have_content(t('doc_auth.headings.interstitial'), wait: 10)
+    wait_for_form_page_load
+  end
+
+  def try_continue_or_submit_images
```

Comment:
@AShukla-GSA - i think we should specify the button we are clicking to make sure the right buttons are appearing when we expect them which itself is a test

Reply:
@amirbey fixed